types/sinon/index.d.ts

```diff
@@ -62,25 +66,25 @@ declare namespace Sinon {
          * Returns true if spy was called at exactly once with the provided arguments.
          * @param args
          */
-        calledOnceWith(...args: MatchArguments<TArgs>): boolean;
-        calledOnceWithExactly(...args: MatchArguments<TArgs>): boolean;
+        calledOnceWith(...args: MatchExactlyArguments<TArgs>): boolean;
```

Comment:
looks like you inadvertently found a bug here...

`calledOnceWith` is meant to be partial, while `calledOnceWithExactly` is exact. 

the former should be using `MatchProvidedArguments` while the latter uses `MatchExactlyArguments`

Reply:
presumably also means we're missing a test for this, otherwise we would've noticed sooner

Reply:
done

Reply:
Done